packages/fuels-test-helpers/src/utils.rs

```diff
@@ -55,10 +55,10 @@ mod tests {
 
     #[tokio::test]
     async fn retry_until_will_respect_delay_between_attempts() -> anyhow::Result<()> {
-        let timestamps_condition_was_called_at: Mutex<Vec<Instant>> = Mutex::new(vec![]);
+        let timestamps_predicate_was_called_at: Mutex<Vec<Instant>> = Mutex::new(vec![]);
```

Comment:
`%s/timestamps/timestamp/g`

Reply:
You renamed the tests but not the actual function argument here, any particular reason?

Reply:
> You renamed the tests but not the actual function argument here, any particular reason?

Yeah, I reverted it by accident trying to be smart with git. I re-reverted. It should be updated by now.

> %s/timestamps/timestamp/g
A fellow VIM user :D 

Actually, these are timestamp**s**, plural. Predicate called more than once, and thus more than one timestamp is written thus we have timestamps of all the times the predicate was called.

If you feel otherwise please re-open.

Reply:
Ah yes my bad! Thought it was only one